src/app/dynamic/enterprise/quotas/template.html

```diff
@@ -0,0 +1,127 @@
+<!--
+               Kubermatic Enterprise Read-Only License
+                      Version 1.0 ("KERO-1.0”)
+                  Copyright © 2022 Kubermatic GmbH
+
+1. You may only view, read and display for studying purposes the source
+   code of the software licensed under this license, and, to the extent
+   explicitly provided under this license, the binary code.
+2. Any use of the software which exceeds the foregoing right, including,
+   without limitation, its execution, compilation, copying, modification
+   and distribution, is expressly prohibited.
+3. THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+   EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+   MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+   IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+   CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+   TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+   SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+END OF TERMS AND CONDITIONS
+-->
+
+<mat-card>
+  <mat-card-header>
+    <mat-card-title fxFlex
+                    fxLayout="column"
+                    fxLayoutAlign="start"
+                    fxLayoutGap="32px">
+      <div fxFlex>Project Quotas</div>
+
+      <div fxLayout="row"
+           fxLayoutAlign="center center">
+        <div class="filters"
+             fxLayout="row"
+             fxLayoutAlign=" center"
+             fxLayoutGap="16px">
+          <div>Filter by</div>
```

Comment:
`Filter by` should be used when there are more than one filter options. Here, we only have a search field so no need for `Filter by`:
![screenshot-localhost_8000-2022 07 12-17_25_11](https://user-images.githubusercontent.com/13975988/178551017-3c8b6284-68e1-4a8e-baf9-7b0a5291f3af.png)


See `Accounts` page for reference:
![screenshot-localhost_8000-2022 07 12-17_25_24](https://user-images.githubusercontent.com/13975988/178551097-e6a7e5b5-a8d4-477f-b694-b7ab3cf779b4.png)

Reply:
"Filter by" removed

Reply:
Search field should have same height like other search fields. For example, accounts search field:
![screenshot-localhost_8000-2022 07 14-00_03_07](https://user-images.githubusercontent.com/13975988/178811720-aa74b2e2-9023-4600-829d-c66aeb623099.png)


![screenshot-localhost_8000-2022 07 14-00_03_42](https://user-images.githubusercontent.com/13975988/178811737-c030f554-0f04-4b49-82d9-a226f51de7fb.png)

Reply:
feedback implemented